<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Hosts</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QTreeView" name="tvHosts">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Vulnerabilities</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QTreeView" name="tvVulns">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Domain Credentials</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QTableWidget" name="tblDomainCreds">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Domain</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Is DA</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Username</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Password</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>lm hash</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ntlm hash</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>http ntlm hash</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Summary</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lblSummary">
          <property name="text">
           <string>Identified 0 hosts, 0 open ports, 0 shells</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QTableWidget" name="tblPortsSummary">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Port</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Number Found</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTableWidget" name="tblVulnsSummary">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Vulnerability</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Number Found</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTableWidget" name="tblDomains">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Domain</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Web Servers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tblWebServers">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>IP Address</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Hostname</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Port</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>URL</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Title</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Pending Work</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="1" column="1">
         <widget class="QTableWidget" name="tblPendingVulnScanning">
          <column>
           <property name="text">
            <string>Port to check</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Count</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Pending Vuln Scanning</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tblPendingFootprinting">
          <column>
           <property name="text">
            <string>Work</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Count</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Pending Footprinting</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Cred Map</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Valid Creds</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Invalid Creds</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tblValidCreds">
          <column>
           <property name="text">
            <string>Host</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Host Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Domain</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Username</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Password</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTableWidget" name="tblInvalidCreds">
          <column>
           <property name="text">
            <string>Host</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Host Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Domain</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Username</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Password</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>autodane v1.0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Written by Dane Goodwin,</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Security Analyst at SensePost</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="lblEmailIcon">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>emailicon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Research@SensePost.com</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lblEmailIcon2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>emailicon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Dane@SensePost.com</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lblSkypeLogo">
              <property name="text">
               <string>skypelogo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>dane_goodwin</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <widget class="QLabel" name="lblSensePostLogo">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>250</height>
                </size>
               </property>
               <property name="text">
                <string>SP Logo</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>SensePost.com</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnAddHost">
      <property name="text">
       <string>Add Host</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnAddDomain">
      <property name="text">
       <string>Add Domain</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnAddDomainCreds">
      <property name="text">
       <string>Add Domain Creds</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnUpdateUI">
      <property name="text">
       <string>Update UI</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QCheckBox" name="cbxUpdateUI">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Update the UI every second</string>
      </property>
      <property name="text">
       <string>Continuously</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
